### PR TITLE
[ENG-4012] Un-silence deprecation manager-capabilities.modifiers-3-13

### PR DESCRIPTION
**Pull request by brianjgeiger**
-   Ticket: [ENG-4012]
-   Feature flag: n/a

## Purpose

Un-silence the deprecation `manager-capabilities.modifiers-3-13` and fix the resulting problems

## Summary of Changes

1. Unsilence deprecation
2. Upgrade ember-auto-import to v2
3. Upgrade ember-modifiers
4. Upgrade ember-moment and imports
5. Spread moment around all the engines
6. Refactor modifiers to work with new api for ember-modifiers
7. Remove unused modifier
8. Fix usage of side-effect from ember computed to give ember power select a placeholder in institutional dashboard

## Side Effects

If I didn't fix the modifiers right, something may have broken there silently.

## QA Notes

1. Make sure that a modal still works (any modal is fine)
2. Make sure branding still has different colors/logos on things
3. Make sure dates/times display properly
4. Do google events still get sent for scrolling down the logged-out home page?
5. Does the Institutional Dashboard's department filter still work decently

[ENG-4012]: https://openscience.atlassian.net/browse/ENG-4012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,13 +128,13 @@
     "ember-angle-bracket-invocation-polyfill": "^2.0.2",
     "ember-animated": "^0.11.0",
     "ember-aria-tabs": "^4.0.0",
-    "ember-auto-import": "^1.11.2",
+    "ember-auto-import": "^2.6.3",
```
Comment (futa-ikeda) [inline package.json:131]: Huz-

File: package.json [hunk 2]
```diff
@@ -250,6 +250,7 @@
     "toastr": "^2.1.4",
     "tracked-built-ins": "^3.1.0",
     "typescript": "^4.2.3",
+    "webpack": "^5.88.0",
```
Comment (futa-ikeda) [inline package.json:253]: -zah!

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Just a couple minor things. Also, maybe worth a small fireside discussion about modifiers and the new modifier API?

File: ember-cli-build.js [hunk 3]
```diff
@@ -109,6 +104,7 @@ module.exports = function(defaults) {
         },
         'ember-cli-babel': {
             includePolyfill: IS_PROD,
+            enableTypescriptTransform: true,
```
[the remark this answers is not in the document]
Comment (brianjgeiger) [reply ember-cli-build.js]: Looks like no. That was from when I was trying to fix some stuff months ago and I guess I forgot to take it back out.

**Pull request merged**